llvm/test/tools/llvm-objdump/XCOFF/disassemble-symbolize-operands2.ll

```diff
@@ -0,0 +1,62 @@
+; RUN: llc -mtriple=powerpc-ibm-aix-xcoff %s -filetype=obj -o %t
+; RUN: llvm-objdump %t -d --symbolize-operands --no-show-raw-insn \
```

Comment:
add the `-r` option?

Reply:
Added -r option. This shows that the "bl" instruction and the RLD display the same function.